app/models/concerns/exhibit_extension.rb

```diff
@@ -7,4 +7,16 @@ module ExhibitExtension
   included do
     has_one :bibliography_service
   end
+
+  def sync_bibliography
+    SyncBibliographyServiceJob.perform_later(self)
+  end
+
+  def build_bibliography_service
```

Comment:
Just out of curiousity, were you able to determine if the `build_bibliography_service` was already generated for us by `ActiveRecord`? http://guides.rubyonrails.org/association_basics.html#methods-added-by-has-one